### PR TITLE
reduce confusion

### PR DESCRIPTION
**Pull request by andrewq**
Two unrelated changes! Panic!

Something to think about: Would it make more sense to describe the desired state, rather than the actions that should be taken to get there?

### CODE REVIEW

**Comment by mlinnell** [comment]
How about

> I have considered adding automated tests, and added or intentionally did not

I have added automated tests, or intentionally chose not to do so

?

**Comment by andrewq** [comment]
I'll accept that, and make it even simpler.

File: pull_request_template.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ _Describe the contents of the PR, how to manually test it, and provide examples/
...
 - [ ] The primary ticket number is referenced in PR and commits
 - [ ] I have performed a self-review of the code
-- [ ] I have considered adding automated tests and added or deliberately did not
-- [ ] I have updated the ticket status
-- [ ] I have added feature flags wherever applicable
\ No newline at end of file
+- [ ] I have adding automated tests, or intentionally chose not to
+- [ ] I have updated the ticket state, if applicable
+- [ ] I have updated the ticket deployment info, if applicable
+- [ ] I have added feature flags, wherever applicable
```
Comment (farhanalam) [inline pull_request_template.md]: While we're at it, can we have the self-review item as the last in the list, and the ticket state changes right before that?

Comment (andrewq) [reply pull_request_template.md]: I mean, ideally I'd like to see them randomized each time, so that people don't get complacent...

**Pull request merged**